keyboards/kinesis/kint36/config.h

```diff
@@ -0,0 +1,71 @@
+#pragma once
+
+/* USB Device descriptor parameter */
+#undef VENDOR_ID
+#define VENDOR_ID 0x1209
+#undef PRODUCT_ID
+#define PRODUCT_ID 0x345C
+#undef DEVICE_VER
+#define DEVICE_VER 0x0001
+#undef MANUFACTURER
+#define MANUFACTURER "https://github.com/stapelberg"
+#undef PRODUCT
+#define PRODUCT "https://github.com/kinx-project/kint"
+
+/* key matrix size */
+#define MATRIX_ROWS 15
+#define MATRIX_COLS 7
+
+/*
+ * Keyboard Matrix Assignments
+ *
+ * Change this to how you wired your keyboard
+ * COLS: AVR pins used for columns, left to right
+ * ROWS: AVR pins used for rows, top to bottom
+ * DIODE_DIRECTION: COL2ROW = COL = Anode (+), ROW = Cathode (-, marked on diode)
+ *                  ROW2COL = ROW = Anode (+), COL = Cathode (-, marked on diode)
+ *
+ */
+#define MATRIX_ROW_PINS  \
+    {                    \
+        LINE_PIN8,       \
+            LINE_PIN9,   \
+            LINE_PIN10,  \
+            LINE_PIN11,  \
+            LINE_PIN7,   \
+            LINE_PIN16,  \
+            LINE_PIN5,   \
+            LINE_PIN3,   \
+            LINE_PIN4,   \
+            LINE_PIN1,   \
+            LINE_PIN0,   \
+            LINE_PIN2,   \
+            LINE_PIN17,  \
+            LINE_PIN23,  \
+            LINE_PIN21,  \
+    }
+
+#define MATRIX_COL_PINS  \
+    {                    \
+        LINE_PIN18,      \
+            LINE_PIN14,  \
+            LINE_PIN15,  \
+            LINE_PIN20,  \
+            LINE_PIN22,  \
+            LINE_PIN19,  \
+            LINE_PIN6    \
+    }
```

Comment:
Why not use the short notation, such as A5, etc?

Reply:
The short notation is not consistently available.

Take a look at https://www.pjrc.com/teensy/card9a_rev1.pdf, which is shipped with every Teensy 3.6 as a physical card.

It contains the numerical pin numbers for every pin, but only a few pins (mostly on the top side of the Teensy 3.6) are named with `A*`.

The situation is the same with the device symbol in the schematic: https://github.com/kinx-project/kint/blob/master/schematic-v2020-06-30.pdf

Reply:
Thats not really how the mapping works within QMK, the `A*` stuff is Arduino notation. <https://www.pjrc.com/teensy/schematic.html> but minus the `PT` prefix should be used (from memory, sometimes you need to dig into the chips actual datasheet).

Reply:
I tried defining the short pin names in `quantum/config_common.h`, but defining e.g. `C1`, `C2`, `C5`, `C6` results in compilation errors in `lib/chibios-contrib/os/hal/ports/KINETIS/MK66F18/hal_lld.c` and others:

```
lib/chibios-contrib/os/hal/ports/KINETIS/MK66F18/hal_lld.c: In function 'MK66F18_clock_init':
./lib/chibios-contrib/os/hal/ports/KINETIS/LLD/GPIOv1/hal_pal_lld.h:154:3: error: expected identifier before '(' token
  154 |   ((ioline_t)((uint32_t)(port) | ((uint32_t)(pad)<<20)))
      |   ^
./lib/chibios-contrib/os/hal/boards/PJRC_TEENSY_3_6/board.h:296:29: note: in expansion of macro 'PAL_LINE'
  296 | #define LINE_PIN23          PAL_LINE(TEENSY_PIN23_IOPORT, TEENSY_PIN23)
      |                             ^~~~~~~~
quantum/config_common.h:186:20: note: in expansion of macro 'LINE_PIN23'
  186 | #        define C2 LINE_PIN23
      |                    ^~~~~~~~~~
lib/chibios-contrib/os/hal/ports/KINETIS/MK66F18/hal_lld.c:194:10: note: in expansion of macro 'C2'
  194 |     MCG->C2 = MCG_C2_LOCRE0 | MCG_C2_EREFS0 | MCG_C2_RANGE0_SET(2);
      |          ^~

```

Are you sure we need the short names and can’t use the `LINE_PIN*` names from https://github.com/qmk/ChibiOS-Contrib/blob/e3a3a24047717af33b098266e11c2e4e5102206a/os/hal/boards/PJRC_TEENSY_3_6/board.h#L273 ?